packages/browser-wallet/src/popup/pages/Web3ProofRequest/Web3ProofRequest.scss

```diff
@@ -5,15 +5,201 @@
         height: 100%;
     }
 
+    &__statement {
+        margin-top: 25px;
+    }
+
     &__actions {
-        margin: auto rem(20px) 0;
+        height: 115px;
+        margin-top: auto;
+        display: grid;
+        grid-auto-flow: column;
+        column-gap: 7px;
+        padding-top: 35px;
+        padding-left: 32px;
+        padding-right: 32px;
+        box-shadow: 0 0 30px 0 #0003;
     }
 
     &__loading-icon {
         max-height: rem(20px);
     }
 
     &__credential-statement-container {
-        margin-bottom: rem(20px);
+        height: calc(100% - 115px);
+        scrollbar-gutter: stable both-edges;
+        padding-right: 16px;
+        padding-left: 16px;
+        overflow-y: auto;
+    }
+
+    &__selector-icon {
+        border-radius: 50%;
+        width: 40px;
+        height: 40px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-right: 25px;
+    }
+
+    &__selector-icon-image {
+        height: 28.3px;
+        width: 28.3px;
+    }
+
+    &__reject-icon {
+        height: 12px;
+        width: 12px;
+
+        path {
+            fill: $color-neutral-white;
+        }
+    }
+
+    &__reject-button {
+        width: 56px;
+        height: 48px;
+        border-radius: 16px;
+        background: #ab2b2b;
+    }
+
+    &__back-button {
+        width: 112px;
+    }
+
+    &__continue-button {
+        flex-grow: 1;
+    }
+
+    &__description {
+        height: 120px;
+        padding-top: 32px;
+        color: $color-neutral-gray-50;
+        margin: 0;
+    }
+}
+
+.new-button-styling {
+    height: 48px;
+    border-radius: 16px;
+    background: linear-gradient(94.88deg, #005a78 -3.22%, #48a2ae 87.51%);
```

Comment:
All the colors should ideally come from the colors file, but I am not sure if we have that for the gradients.

Reply:
I've removed most literal colors, expect in gradient/shadows